### ISSUE TITLE
Tests not discovered with Expecto >= 4.x.x

### ISSUE THREAD

**Issue by fbehrens** [adamchester/expecto-adapter#11]
Hi,

I addedd it through `nuget Expecto.VisualStudio.TestAdapter version_in_path: true `
`paket install`, 
This installs `Expecto.VisualStudio.TestAdapter (1.0.0-alpha005) - version_in_path: true`

and then restart Visual Studio.
```
Microsoft Visual Studio Enterprise 2015
Version 14.0.25431.01 Update 3
Microsoft .NET Framework
Version 4.6.01055
```
I can open Test-Explorer bun none of my Expecto tests are discovered.
It's not that I really wonder about something in Visual Studio is not working, 
but it would be nice if it would.

**Comment by nikolamilekic** [comment]
@fbehrens Try with an older version of Expecto (3.2). Also, alpha005 does not support FsCheck (issue #9) and async tests (issue #4).

**Comment by adamchester** [comment]
Released [1.0.0-alpha006](https://github.com/adamchester/expecto-adapter/releases/tag/1.0.0-alpha006) however I expect this issue still exists.

**Comment by fbehrens** [comment]
I tried with 
```
Main Expecto - 3.2.1
Main Expecto.VisualStudio.TestAdapter - 1.0.0-alpha006
```
ropenend solution, rebuild solution, but no test in test-Explorer apperar.
With `expecto 3.2.0` i got the same result.

Here is my example  code https://github.com/fbehrens/Fsharp-playground

**Comment by nikolamilekic** [comment]
@fbehrens You need to add Expecto.VisualStudio.TestAdapter to your project's paket.references (followed by .paket/paket.exe install). I just tried that with your code and it worked.

**Comment by fbehrens** [comment]
@nikolamilekic Thanks for support :). It discoverd the test in with Expecto `3.2.1` after i did. 
@adamchester Thanks for the update, I can confirm your expectations.

**Comment by fbehrens** [comment]
I don't know if you will also support VS2017. Will You ?
Right now tests are not discovered in VS2017.
```
Main Expecto - 3.2.1
Main Expecto.VisualStudio.TestAdapter - 1.0.0-alpha006

Microsoft Visual Studio Enterprise 2017
Version 15.0.26228.4 D15RTWSVC
Microsoft .NET Framework
Version 4.6.01586
```

**Comment by nikolamilekic** [comment]
@fbehrens That's strange because I tried your project with 2017 and it worked after I made the paket.references change.

**Comment by fbehrens** [comment]
@nikolamilekic Yes, indeed. I have no idea why. 
Its a fresh install of `VS2017`. Just cloned, `paket restore`, and rebuild all.

**Comment by adamchester** [comment]
Hey @fbehrens, your solution currently works here in 2017 too, no changes required.
![image](https://cloud.githubusercontent.com/assets/570470/23827010/4d4729e4-06f5-11e7-91e4-2865fdbfecb0.png)

I also tried upgrading expecto to 4.0.3 and it fails still, so I am leaving this open:
![image](https://cloud.githubusercontent.com/assets/570470/23827054/4e59efaa-06f6-11e7-9cb8-c2b925af4ec3.png)

**Comment by adamchester** [comment]
Alright, the solution here might as simple as recompiling this test adapter against expecto 4.x, fixing a few minor breaking changes, then publishing again.

![image](https://cloud.githubusercontent.com/assets/570470/23827376/2315a264-06fe-11e7-8f7c-3fde7774ea0c.png)

**Comment by adamchester** [comment]
Fixed with https://github.com/adamchester/expecto-adapter/releases/tag/4.0.0-alpha007

**Comment by fbehrens** [comment]
@nikolamilekic Thanks for checking with VS2017. I pulled from the wrong remote and missed again the entry in the `paket.references` :facepalm:

@adamchester thanks for the fixes and update to `4.0.3`
